src/docs/WSL_BUILD.md

```diff
@@ -0,0 +1,25 @@
+# Local development build [Ubuntu Linux / WSL2]
+
+Make sure you have the dependencies.
+- Install Python 
+```
+sudo apt update
+sudo apt install python3 python3-pip
```

Comment:
You install pip, but not platformio. We need platformio to run build-for-machine.

Reply:
I would suggest installing this in a virtual env, but that may be more personal preference. I don't like my projects messing with the global python path and the packages in there. Python gets used all over my OS, so I don't want anything affected by some dep installed by platformio. I am running ubuntu natively on my laptop.

```
virtualenv -p python3 .venv
source .venv/bin/activate
pip install platformio
```
And then anytime you start a new session:
```
source .venv/bin/activate
```

Reply:
Isn't platformio installed automatically from the build script? Let me check that...

Reply:
It would appear that you are right... again. :) I'll add the necessary changes to the doc.